### ISSUE TITLE
Read only dashboard

### ISSUE THREAD

**Issue by dominicbr** [tsumaru720/conspectus#29]
I'd like the ability to host a read-only copy of the dashboard.

The read-only version would (ideally) present all of the same data, just remove the ability for the viewer to add any new data points.

Ideally this would be achieved through an auth mechanism of some sort, but I'm not opposed to hosting a separate "Read-only" instance of the dashboard either.

**Comment by tsumaru720** [comment]
Yeah, I've thought about this feature myself for the Demo, but dont really want to use my own data in a public fashion. 

I want to avoid Auth logic for the time being, so for now I'd probably do this via a config switch so you can just run a separate instance but with the same database.

(Although at this point, whats the difference with just using a Read only MySQL user)

**Comment by tsumaru720** [comment]
We've added support for being read only by an env var `CONSPECTUS_READ_ONLY`
